routes/favicon.coffee

```diff
@@ -0,0 +1,16 @@
+request = require 'request'
+
+url = null
+
+module.exports = (req, res, next) ->
+	if url?
+		res.redirect 301, url
+		return this
```

Comment:
This should actually be analogous to the return value of the other (formerly `if`) branch, which has no explicit return statement, so a plain `return` is the correct outcome. The commit was changed.